light_curves/light_curve_classifier.md

```diff
@@ -63,6 +63,9 @@ Trained classifiers as well as estimates of their accuracy and plots of confusio
 - `scipy` for statistical analysis
 - `json` for storing intermediate files
 - `google_drive_downloader` to access files stored in google drive
+
+## Runtime
+This notebook takes ~170s to run to completion on Fornax using the 'Astrophysics Default Image' and the 'Large' server with 16GB RAM/ 4CPU. 
```

Comment:
Maybe add as "As of 2024 Aug", or some other date indicator.

Reply:
done